### ISSUE TITLE
Come up with a method other than neighborhoods for dividing a city

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1910]
Although neighborhoods sound nice in theory, there are some major problems with actually using them for Project Sidewalk. A neighborhood isn't really something that is rigidly defined. Neighborhoods often overlap with each other. They are often not something that is official, so there aren't fancy GIS maps out there for us to use. Sometimes some part of a city has well defined neighborhoods, and other parts of the city do not.

So whenever I have to create a database for a new city, the hardest part is the neighborhood division. Finding an option other than neighborhoods might make bringing Project Sidewalk to new cities easier. Alternatively, we might just want to come up with a set of guidelines and instructions for how to wade through the messy landscape of neighborhoods when creating the database for a new city. 

An example: I was looking at potentially creating a database for Columbus OH. This is what ["Neighborhoods in Columbus" Wikipedia](https://en.wikipedia.org/wiki/Neighborhoods_in_Columbus,_Ohio) article says.
>Columbus, Ohio does not have defined neighborhoods, per the city of Columbus, making their areas difficult to define. Neighborhoods often overlap and vary in size. Many of Columbus's larger neighborhoods integrate smaller neighborhoods. Additionally, modern interpretations of neighborhood borders vary significantly as historical neighborhoods, villages, towns and townships have been annexed and absorbed by the city of Columbus. There are Area Commissions that exist, but these generally are larger than neighborhoods themselves and typically incorporate smaller neighborhoods.

**Comment by jonfroehlich** [comment]
Right, I agree. But we have to divide a city somehow and, if possible, each city already has some sort of sub-division system (e.g., neighborhoods, areas, etc.). I also think our "neighborhoods" in SPGG are too big. The task to complete Del Valle feels enormous because the neighborhood is 50+ miles long. I think we should break up larger "areas" into 10-20 mile chunks. The easiest way to do this would be to use cardinal directions and split into quadrants using an X-split (North Del Valle, South Del Valle, East Del Valle, West Del Valle) or into eights (Northeast Del Valle, North Del Valle, Northwest Del Valle, etc.).

**Comment by misaugstad** [comment]
I've also really come around on this after setting up a few cities. We don't really want to set up in a new city unless we have decent buy in from people in that city. Which almost always means that we have people who can help us pick an appropriate definition of "neighborhoods" for their city. I think it's okay that neighborhoods are not rigidly defined and that we have conversations with local residents about what the divisions should look like. Those conversations are probably the most appropriate way to do this.

**Comment by misaugstad** [comment]
I wanted something formulaic and possibly automated, but I think that this is just a piece that _should_ be manual.

**Comment by misaugstad** [comment]
Closing as answered by the discussion itself: the thread landed on "this should stay manual." We only launch in a city where we have real local buy-in, which means we have partners who can tell us what the meaningful divisions are — and that conversation is the right way to pick them, even though neighborhoods aren't rigidly defined.

The piece that is still actionable — letting a city have more than one kind of division (wards, census blocks, etc.) rather than a single neighborhood layer — is tracked in #3489, gated on #3488.
